src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -159,7 +159,7 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
               |'asset name' as name,
               |null as parentId,
               |'asset description' as description,
-              map("foo", null, "bar", "test") as metadata,
+              |map("foo", null, "bar", "test") as metadata,
```

Comment:
Strange, was this working earlier?

Reply:
Not sure what is happening here. The test was failing when I came in this morning, so I looked at it and saw the pipe wasn't there, then I added the pipe and the test started passing. It's also missing on the master branch, although it built successfully anyway.

Reply:
Both work, the output just gets ugly without the pipe, but it's still syntactically valid SQL. Most likely a random test failure? Either way it should have the pipe, so nice catch.

```
@ val assetsTestSource = "asdf"
assetsTestSource: String = "asdf"
@ s"""
                |select "1" as externalId,
                |'asset name' as name,
                |null as parentId,
                |'asset description' as description,
                |map("foo", null, "bar", "test") as metadata,
                |'$assetsTestSource' as source,
                |bigint(10) as id
        """.stripMargin
res1: String = """
select "1" as externalId,
'asset name' as name,
null as parentId,
'asset description' as description,
map("foo", null, "bar", "test") as metadata,
'asdf' as source,
bigint(10) as id
      """
@ s"""
                |select "1" as externalId,
                |'asset name' as name,
                |null as parentId,
                |'asset description' as description,
                map("foo", null, "bar", "test") as metadata,
                |'$assetsTestSource' as source,
                |bigint(10) as id
        """.stripMargin
res2: String = """
select "1" as externalId,
'asset name' as name,
null as parentId,
'asset description' as description,
              map("foo", null, "bar", "test") as metadata,
'asdf' as source,
bigint(10) as id
      """
```